juniper/src/executor/look_ahead.rs

```diff
@@ -100,7 +100,6 @@ where
 #[derive(Debug, Clone, PartialEq)]
 pub struct ChildSelection<'a, S: 'a> {
     pub(super) inner: LookAheadSelection<'a, S>,
-    pub(super) applies_for: Applies<'a>,
```

Comment:
This change makes `ChildSelection` completely redundant. I looked at the code and haven't seen using it outside this module. I guess it could be stripped out completely.

Reply:
You are right. I'll remove it.